dmpr.py

```diff
@@ -63,28 +65,28 @@ def log(self, tracepoint, msg):
 
 
 class DMPR(object):
-
+    # TODO: change signature, log is required
     def __init__(self, log=None, tracer=None):
-        assert(log)
+        assert (log)
```

Comment:
sure? Whitespace before parentheses?

Reply:
parentheses is not even needed, this was just from the first round of auto pep8, will be obsolete soon anyway